### PR TITLE
Open cluster detail on click on table row

### PR DESCRIPTION
**Pull request by ekhomyakova**

### CODE REVIEW

**Comment by naumvd95** [comment]
what row?
pls pls pls pls pls provide screenshots or valid description
@ekhomyakova

**Comment by ekhomyakova** [comment]
@naumvd95 when you click on the table row of Running Clusters, cluster details are opened

File: kqueen_ui/blueprints/ui/templates/ui/partial/_cluster_row.html [hunk 1]
```diff
@@ -1,44 +1,58 @@
...
+  {% if cluster.metadata %}
+  <tr id="metadata-cluster-{{ index }}">
+    <td colspan="5" style="padding: 0px; border-top: 0px;">
+      <div style="margin: 8px;" id="cluster-{{ index }}" class="collapse">
+        <table class="table table-hover no-actions" id="table-clustermeta">
+          <tbody>
+            {% for key, value in cluster.metadata.items() %}
+            <tr>
+              <th style="width: 300px;">{{ key }}</th>
+              <td>{{ value|urlize(40, target="_blank") }}</td>
```
Comment (katyafervent) [inline kqueen_ui/blueprints/ui/templates/ui/partial/_cluster_row.html]: value can be list or dictionary as well. Could you please resolve this case also?)

Comment (ekhomyakova) [reply kqueen_ui/blueprints/ui/templates/ui/partial/_cluster_row.html]: Only if you allow me to refactor quotes, indentation and line breaks in the same commits with the main request  :/

Comment (katyafervent) [reply kqueen_ui/blueprints/ui/templates/ui/partial/_cluster_row.html]: I thought it is new code, it is hard to catch what is changed in _cluster_row.html due to refactoring again. I'm not able to review it

**Pull request merged**